qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -495,9 +495,7 @@ def run(self, *args, **kwargs) -> None:  # type: ignore[no-untyped-def]
 
     def get_translation_stage_plugin(self) -> str:
         """Return the default translation stage plugin name for IBM backends."""
-        if not self.options.use_fractional_gates:
-            return "ibm_dynamic_circuits"
-        return "ibm_fractional"
+        return "ibm_dynamic_and_fractional"
```

Comment:
If fractional gates can still be disabled (correct me if I'm wrong), I think that it still make senses to keep the `ibm_dynamic_circuits` standalone plugin, right? So having something like:

```suggestion
        if not self.options.use_fractional_gates:
                return "ibm_dynamic_circuits"
        return "ibm_dynamic_and_fractional"
```

In this case I would "undeprecate" the ibm_dynamic_circuits.